src/Symfony/Bundle/TwigBundle/Resources/config/twig.php

```diff
@@ -73,6 +73,7 @@
             ->call('setDebug', [param('kernel.debug')])
             ->call('setTokenStorage', [service('security.token_storage')->ignoreOnInvalid()])
             ->call('setRequestStack', [service('request_stack')->ignoreOnInvalid()])
+            ->call('setStrictVariables', [param('twig.strict_variables')])
```

Comment:
The default is `strict_variables: '%kernel.debug%'`
Which means we're loosening the checks in prod mode, but that won't solve anything in dev mode.
Or do you plan to turn the setting to false in both prod/dev envs in your use case?

Actually, reading the issue and the PR again, I think we're missing a "why". What do you want to achieve with this change?